config/opal_config_libevent.m4

```diff
@@ -70,6 +70,23 @@ AC_DEFUN([OPAL_CONFIG_LIBEVENT], [
     AS_IF([test "$external_libevent_happy" = "0" -a "$internal_libevent_happy" = "0"],
           [AC_MSG_ERROR([Could not find viable libevent build.])])
 
+    AS_IF([test "$external_libevent_happy" = "1"],
+        [AS_IF([test -n "$with_libevent"],
```

Comment:
why is this conditional block different than hwloc?

Reply:
ompi requires libevent_core and libevent_pthreads, as a pmix dependency I believe. From what I can tell just checking libevent.pc only returns -levent.

Reply:
can you add a comment to that effect (that we need -llibevent_core (the new hotness), but the .pc file only returns -levent (the old brokenness), so the next person doesn't have to figure that out?